connection.go

```diff
@@ -654,13 +664,15 @@ func (c *Connection) queueFrame(q *[]frame, f frame) {
 func (c *Connection) queueStreamFrames(pt uint8, protected bool, bareAcks bool) (int, error) {
 	c.log(logTypeConnection, "%v: sendQueuedStreamData pt=%v, protected=%v",
 		c.label(), pt, protected)
-	left := c.mtu
 	frames := make([]frame, 0)
 	sent := int(0)
 	acks := c.recvd.prepareAckRange(protected, false)
 	now := time.Now()
 	txAge := time.Duration(c.retransmitTime) * time.Millisecond
 
+	aeadOverhead :=  c.determineAead(pt).Overhead()
+	left := c.mtu - aeadOverhead - kLongHeaderLength // TODO: check header type
```

Comment:
Can you save this so you don't need to compute it again? Then you won't need aeadOverhead either.

Reply:
Also, please fix TODO format.